app/models/defendant.rb

```diff
@@ -77,6 +87,12 @@ def offence_details
     details.flat_map { |detail| detail["offences"] }.group_by { |offence| offence["id"] }
   end
 
+  def judicial_results_array
+    return [] if details.blank?
+
+    details.flat_map { |detail| detail["judicialResults"] }.uniq.compact
```

Comment:
Do we know that we want these to be unique this time? I don't know the domain well enough to know why pleas was an array of unique elements...

Reply:
I think that's just in case judicial results are repeated for a given defendant. This makes sure we end up with a list of uniques.